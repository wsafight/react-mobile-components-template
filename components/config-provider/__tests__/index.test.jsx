import React from 'react';
import { render } from 'enzyme';
import toJson from 'enzyme-to-json';
import LocaleProvider from '../index';
import Alert from '../../alert';
import localeCN from '../locale/zh_CN';
import localeEN from '../locale/en_US';

describe('LocaleProvider', () => {
  it('renders correctly with zh_CN', () => {
    const wrapper = render(
      <LocaleProvider locale={localeCN}>
        <Alert />
      </LocaleProvider>,
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('renders correctly with en_US', () => {
    const wrapper = render(
      <LocaleProvider locale={localeEN}>
        <Alert />
      </LocaleProvider>,
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
