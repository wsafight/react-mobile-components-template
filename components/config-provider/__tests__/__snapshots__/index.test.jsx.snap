// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocaleProvider renders correctly with en_US 1`] = `
<div
  class="za-alert za-alert--radius"
/>
`;

exports[`LocaleProvider renders correctly with zh_CN 1`] = `
<div
  class="za-alert za-alert--radius"
/>
`;
